frontend/zc_messaging/src/redux/services/messages.js

```diff
@@ -14,32 +14,36 @@ export const messagesApi = createApi({
   endpoints: builder => ({
     getMessagesInRoom: builder.query({
       async queryFn(_arg, _queryApi, _extraOptions, fetchWithBQ) {
-        const { orgId, roomId } = _arg
+        const { orgId, roomId, pageIndex } = _arg
         const getMessagesInRoomResponse = await fetchWithBQ(
-          `/org/${orgId}/rooms/${roomId}/messages`
+          `/org/${orgId}/rooms/${roomId}/messages?page=${pageIndex}&size=15`
         )
-        if (Array.isArray(getMessagesInRoomResponse?.data?.data?.data)) {
+        if (Array.isArray(getMessagesInRoomResponse?.data?.data.data)) {
```

Comment:
Leave that ? there. It's a fail safe for when the messages fail to show up. Otherwise the app will throw an error and may dismount

Reply:
Okay